xyz-hub-service/src/main/resources/config.json

```diff
@@ -2,7 +2,7 @@
   "INSTANCE_COUNT": 1,
   "HOST_NAME": "127.0.0.1",
   "HTTP_PORT": 8080,
-  "ADMIN_MESSAGE_PORT": 8181,
+  "ADMIN_MESSAGE_PORT": 8080,
```

Comment:
only valid when fut.complete can be executed by changing the code on my previous comment.